tests/cpp/parser_tests.cpp

```diff
@@ -6479,6 +6479,94 @@ rule cuckoo_desktop
 	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
 }
 
+TEST_F(ParserTests,
+CuckooClassCreatedModuleFunction) {
+	prepareInput(
+R"(
+import "cuckoo"
+
+rule cuckoo_class_created
+{
+	condition:
+		cuckoo.process.class_created(/SomeEvilClass/)
+}
+)");
+
+	EXPECT_TRUE(driver.parse(input));
+	ASSERT_EQ(1u, driver.getParsedFile().getRules().size());
+
+	const auto& rule = driver.getParsedFile().getRules()[0];
+	EXPECT_EQ(R"(cuckoo.process.class_created(/SomeEvilClass/))", rule->getCondition()->getText());
+
+	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
+}
+
+TEST_F(ParserTests,
+CuckooClassSearchedModuleFunction) {
+	prepareInput(
+R"(
+import "cuckoo"
+
+rule cuckoo_class_searched
+{
+	condition:
+		cuckoo.process.class_searched(/SomeEvilClass/)
+}
+)");
+
+	EXPECT_TRUE(driver.parse(input));
+	ASSERT_EQ(1u, driver.getParsedFile().getRules().size());
+
+	const auto& rule = driver.getParsedFile().getRules()[0];
+	EXPECT_EQ(R"(cuckoo.process.class_searched(/SomeEvilClass/))", rule->getCondition()->getText());
+
+	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
+}
+
+TEST_F(ParserTests,
+CuckooWindowCreatedModuleFunction) {
+	prepareInput(
+R"(
+import "cuckoo"
+
+rule cuckoo_window_created
+{
+	condition:
+		cuckoo.process.window_created(/SomeEvilWindow/)
+}
+)");
+
+	EXPECT_TRUE(driver.parse(input));
+	ASSERT_EQ(1u, driver.getParsedFile().getRules().size());
+
+	const auto& rule = driver.getParsedFile().getRules()[0];
+	EXPECT_EQ(R"(cuckoo.process.window_created(/SomeEvilWindow/))", rule->getCondition()->getText());
+
+	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
+}
+
+TEST_F(ParserTests,
+CuckooWindowSearchedModuleFunction) {
+	prepareInput(
+R"(
+import "cuckoo"
+
+rule cuckoo_window_searched
+{
+	condition:
+		cuckoo.process.window_searched(/SomeEvilWindow/)
+}
+)");
+
+	EXPECT_TRUE(driver.parse(input));
+	ASSERT_EQ(1u, driver.getParsedFile().getRules().size());
+
+	const auto& rule = driver.getParsedFile().getRules()[0];
+	EXPECT_EQ(R"(cuckoo.process.window_searched(/SomeEvilWindow/))", rule->getCondition()->getText());
+
+	EXPECT_EQ(input_text, driver.getParsedFile().getTextFormatted());
+}
+
```

Comment:
In all these tests, we use the `driver` instance of `ParserDriver` that has been constructed with default parameter `ImportFeatures::AllCurrent`.
Maybe we would also like to test, that the behavior would be the same if we used an instance of `ParserDriver` class constructed with `ImportFeatures::Avast`, and, that when we would use `ImportFeatures::VirusTotal`, the parsing would fail.